app/components/SocialMediaBanner.tsx

```diff
@@ -2,52 +2,88 @@ import React from "react";
-        <div className="col-span-1 ..">
-          <div className="text-4xl flex flex-row min-h-[50%] justify-center items-center text-center">
-            <h1>Follow our story on social media!</h1>
-          </div>
-          <div className="text-md text-center">
-            <p>
-              Join us in our mission to transform musculoskeletal care for all,
-              one child at a time,
-            </p>{" "}
-            <p>through education, research, and unwavering support.</p>
-          </div>
+      </div>
+      <div className="hidden md:block col-span-1">
+        <div className="text-4xl flex flex-row min-h-[50%] justify-center items-center text-center">
+          <h1>Follow our story on social media!</h1>
         </div>
-        <div className="row-span-1  .. py-4 md:py-0">
-          <div className="flex flex-row min-h-[75%] justify-center items-center space-x-4 ..">
-            <a href="https://www.instagram.com/orthokids_losangeles/ ..">
-              <Image src={FB} alt="Facebook Logo" />
-            </a>
-            <a href="https://www.linkedin.com/school/luskin-orthopaedic-institute-for-children/">
-              <Image src={LinkedIN} alt="LinkedIn Logo" />
-            </a>
-            <a href="https://twitter.com/i/flow/login?redirect_after_login=%3FLuskinOIC">
-              <Image src={TwitterX} alt="X Logo" />
-            </a>
-            <a href="https://www.youtube.com/channel/UC3JqUFKs4TmQUGa9ekOYePA">
-              <Image src={Youtube} alt="Youtube Logo" />
-            </a>
-            <a href="https://www.yelp.com/biz/luskin-orthopaedic-institute-for-children-los-angeles">
-              <Image src={Yelp} alt="Yelp Logo " />
-            </a>
-          </div>
+        <div className="flex items-center justify-center text-md text-center">
+          <p className="w-5/6">
+            Join us in our mission to transform musculoskeletal care for all,
+            one child at a time, through education, research, and unwavering
+            support.
+          </p>
         </div>
       </div>
+      <SocialMediaIcons />
+    </div>
+  );
+}
+
+function MobileBanner() {
+  return (
+    <div className="block md:hidden min-h-[75%]">
+      <div className="flex flex-row items-center justify-between">
+        <Title1 className="pl-10 pt-4">
+          Follow our story on social media!
+        </Title1>
+        <Image src={OzzieMobile} alt="Ozzie" />
+      </div>
+      <Text1 className="px-10">
+        Join us in our mission to transform musculoskeletal care for all, one
+        child at a time, through education, research, and unwavering support.
+      </Text1>
+      <SocialMediaIcons />
+    </div>
+  );
+}
+
+function SocialMediaIcons() {
+  return (
+    <div className="row-span-1 .. pt-8 pb-10 md:pt-0 md:pb-0 px-10 md:px-3">
+      <div className="flex flex-row min-h-[75%] justify-center items-center space-x-4 ..">
+        <a href="https://www.instagram.com/orthokids_losangeles/ ..">
```

Comment:
Oops! broken link for facebook, should be this: https://www.facebook.com/LuskinOIC/

Reply:
Fixed! I also noticed the X link (https://twitter.com/i/flow/login?redirect_after_login=%3FLuskinOIC) forces the user to log in every time, so I replaced it with this: https://twitter.com/LuskinOIC